examples/bratko_kopec/bratko_kopec.py

```diff
@@ -26,6 +26,7 @@ def test_epd(engine, epd, VariantBoard, movetime):
 
     engine.ucinewgame()
     engine.setoption({"UCI_Variant": VariantBoard.uci_variant})
+    engine.setoption({"Threads": threads})
```

Comment:
You can use a single `engine.setoption` here. Example: https://python-chess.readthedocs.io/en/latest/variant.html#uci